tests/unit/src/test/scala/tests/rangeFormatting/MultilineStringRangeFormattingWhenSelectingSuite.scala

```diff
@@ -110,23 +110,23 @@ class MultilineStringRangeFormattingWhenSelectingSuite
     s"""
        |object Main {
```

Comment:
The change here is due to the new way of calculating expected indent, I think we should at some point detect whether we are pasting or selecting, but not sure exactly how to do that.